test/JDBC/input/constraint_table_usage.sql

```diff
@@ -0,0 +1,51 @@
+use master;
```

Comment:
We can remove this test file as we already have tests in prepare and verify format. This just makes a redundant test file.